projboard/models.py

```diff
@@ -52,6 +52,67 @@ def get_user_by_nickname(nickname):
             return None
         return user
 
+    @staticmethod
+    def create_user(email, password, name, nickname):
```

Comment:
Since the method is not only creating a new User, but also checks if the user already exists and return it, I would rename the method's name to `get_user`

Reply:
+1

Reply:
Done